test/NameBinding/Inputs/property_wrappers_A.swift

```diff
@@ -0,0 +1,8 @@
+@propertyWrapper
+public struct Wrapper<Value> {
+	public var wrappedValue: Value
```

Comment:
Funny indentation

Reply:
Ugh, annoying. I'll clean it up in a follow-up